src/main/java/se/sundsvall/billingpreprocessor/service/util/ProblemUtil.java

```diff
@@ -0,0 +1,15 @@
+package se.sundsvall.billingpreprocessor.service.util;
+
+import static org.zalando.problem.Status.INTERNAL_SERVER_ERROR;
+
+import org.zalando.problem.Problem;
+import org.zalando.problem.ThrowableProblem;
+
+public class ProblemUtil {
+	private ProblemUtil() {}
+
+	public static ThrowableProblem createProblem(String message) {
+		return Problem.valueOf(INTERNAL_SERVER_ERROR, message);
+	}
```

Comment:
```suggestion
	public static Supplier<ThrowableProblem> createProblem(String message) {
		return () ->  Problem.valueOf(INTERNAL_SERVER_ERROR, message);
	}
```
Just a style suggestion, using this you can remove the supplier in every createProblem call.

Reply:
Nice